src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/02-linux-installation.md

```diff
@@ -0,0 +1,239 @@
+
+```bash
+skm linux install
+```
+
+![Terminal](/splashkit/linux-installation-fig6.png)
+
+This command may or may not require you to enter your password to install additional dependencies for SplashKit.
+
+![Terminal](/splashkit/linux-installation-fig7.png)
+
+### skm Global Install
+
+To install SplashKit globally, run the following command:
+
+```bash
+skm global install
+```
+
+![Terminal](/splashkit/linux-installation-fig8.png)
+
+## Installing Visual Studio Code
+
+Next you will need to install visual studio code, if you have this already installed, then move to installing the necessary extensions step.
+
+### Download Visual Studio Code
+
+ To install Visual Studio Code, go to the [Visual Studio Code website](https://code.visualstudio.com/) and download the `.deb` file.
+
+![Terminal](/splashkit/linux-installation-fig9.png)
+
+![Terminal](/splashkit/linux-installation-fig10.png)
+
+### Install Visual Studio Code
+
+To install Visual Studio Code, move to the directory where the downloaded `.deb` file by doing:
+
+```bash
+cd ~/Downloads
+```
+
+Then run the following command:
+
+```bash
+sudo dpkg -i code_1.*.deb
+```
+
+![Terminal](/splashkit/linux-installation-fig11.png)
+
+## Installing Necessary Extensions for Visual Studio Code
+
+To install the necessary extensions, keep the terminal open and run the following commands:
+
+### C# Extension
+
+[C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp):
+
+```bash
+code --install-extension ms-dotnettools.csharp
+```
+
+![Terminal](/splashkit/linux-installation-fig12.png)
+
+### C# Dev Kit
+
+[C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit):
+
+```bash
+code --install-extension ms-dotnettools.csdevkit
+```
+
+![Terminal](/splashkit/linux-installation-fig13.png)
+
+### Intellicode for C# Dev Kit
+
+[Intellicode for C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscodeintellicode-csharp):
+
+```bash
+code --install-extension ms-dotnettools.vscodeintellicode-csharp
+```
+
+![Terminal](/splashkit/linux-installation-fig14.png)
+
+### C++ Extension Pack
+
+[C++ Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack):
+
+```bash
+code --install-extension ms-vscode.cpptools-extension-pack
+```
+
+![Terminal](/splashkit/linux-installation-fig15.png)
+
+### Installing .net SDK
+
+To install the .net SDK, head to the following site and select for your operating system.
+
+[.NET SDK](https://dotnet.microsoft.com/en-us/download?initial-os=linux)
+
+![Terminal](/splashkit/linux-installation-fig16.png)
+
+![Terminal](/splashkit/linux-installation-fig17.png)
+
+Then to install the package, first head to your downloads folder again, then run the following command:
+
+```bash
+sudo apt-get update && \
+  sudo apt-get install -y dotnet-sdk-8.0
+```
+
+![Terminal](/splashkit/linux-installation-fig18.png)
+
+## RESTART Your Computer
```

Comment:
No need for all caps on RESTART.

Reply:
I think its necessary to ensure they see it.

Reply:
The words are already highlighted as part of a title. Any reasons why restarting the terminal is critical should rather be highlighted and emphasised within the content.

Reply:
Dot net will not work unless the computer has been restarted.